src/test/java/com/wikia/webdriver/testcases/desktop/articlecrudtests/ArticleSourceModeTests.java

```diff
@@ -280,4 +287,57 @@ public void RTE_019_MoreSympolsTools() {
     source.checkSymbolsTools();
     source.submitArticle();
   }
+
+  @Test(groups = {"RTE_draft","draft_saving"})
+  @Execute(asUser = User.SUS_REGULAR_USER3, onWikia = "draftsavetest")
+  public void RTE_draft_intervening_accepted() {
+    String articleName = PageContent.ARTICLE_NAME_PREFIX + DateTime.now().getMillis();
+//    String articleName2 = articleName+"D";
+
+    SourceEditModePageObject source = new SourceEditModePageObject().openArticle(articleName);
+    source.addContentInSourceMode(articleName);
+//    source.focusTextArea();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)